### ISSUE TITLE
Composer dependency issue with alom/graphviz

### ISSUE THREAD

**Issue by AntonStoeckl** [Trismegiste/Mondrian#38]
Hi there,

you depend on alom/graphviz 0.9.\* but alom/graphviz only has versions 1.0.0 and 1.0.1

Best regards, Anton

**Comment by Trismegiste** [comment]
I use my own fork of alom/graphviz, it is in the composer.json

**Comment by AntonStoeckl** [comment]
Hm, I think this only works if someone clones your repo.
But I have added it as a dependency to an existing project, in this case it does not use the repo settings from your composer.json.
I think it's solvable by adding the repo for your alom/graphviz to repositories in "my" project, though.
I'll try that to play around with it, thx!

**Comment by nenadalm** [comment]
:+1:

**Comment by lenton** [comment]
Yes, I'm getting the same issue when doing a `composer global require trismegiste/mondrian`:

```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Installation request for trismegiste/mondrian dev-master -> satisfiable by trismegiste/mondrian[dev-master].
    - trismegiste/mondrian dev-master requires alom/graphviz 0.9.* -> no matching package found.
```

As @AntonStoeckl stated, doing a `git clone` and `composer install` works fine. Trying to install it as a project dependancy or globally fails because composer doesn't support repositories recursively: https://getcomposer.org/doc/faqs/why-can%27t-composer-load-repositories-recursively.md. You therefore have to add the custom repository in your project's `composer.json` for it to work.

To fix this I'd suggest we switch to using @Trismegiste fork directly (`trismegiste/graphviz`), until we can upgrade to using `alom/graphviz` version `1.0.0`.
